docs/getting-started/deno.md

```diff
@@ -213,6 +213,14 @@ deno test hello.ts
   }
 }
 ```
+To use middleware you need to use the [Deno directory](https://docs.deno.com/runtime/fundamentals/configuration/#custom-path-mappings) syntaxt in the import
+```json
+{
+  "imports": {
+    "hono/": "npm:/hono/"
```

Comment:
This means you can't use `hono/logger` if it's not specified?

```ts
import { logger } from 'hono/logger'
```

I tried it, but it works without the specified. My `deno.json` is

```json
{
  "imports": {
    "hono": "npm:hono"
  },
  "tasks": {
    "start": "deno run --allow-net main.ts"
  },
  "compilerOptions": {
    "jsx": "precompile",
    "jsxImportSource": "hono/jsx"
  }
}
```